File: aula05/ex2.py
notas = ["","","","",""]
soma = 0
cont = 0
for x in range(5):
    notas[x] = float(input("digite a nota:"))
for z in range(5):
    soma += notas[z]
media = soma/5
for y in range(5):
    if notas[y] >= media:
        cont = cont + 1
print("media da sala:", media,"alunos aprovados:", cont)

File: aula 4 java/ex2.java
package aula4;
import java.util.Scanner;
public class ex2 {

	public static void main(String[] args) {
		float nota[] = new float[5];
		 float media = 0, soma = 0;
		Scanner entrada = new Scanner(System.in);
		
		for (int x = 0; x< nota.length; x++) {
			System.out.println("Digite as notas dos alunos:");
			nota[x] = entrada.nextInt();
		}
		for (int x = 0; x< nota.length; x++) {
			soma = soma + nota[x];
		}
		media = soma/nota.length;
		System.out.println(media);
	}
}